pg_search/src/api/mod.rs

```diff
@@ -295,3 +295,25 @@ pub struct OrderByInfo {
     pub feature: OrderByFeature,
     pub direction: SortDirection,
 }
+
+pub trait ToTantivyJson {
+    fn key(&self) -> String;
+    fn json_value(&self) -> serde_json::Value;
+}
+
+impl ToTantivyJson for OrderByInfo {
+    fn key(&self) -> String {
+        "order".to_string()
+    }
+
+    fn json_value(&self) -> serde_json::Value {
+        match self.feature {
+            OrderByFeature::Field(_) => match self.direction {
+                SortDirection::Asc => serde_json::json!({ "_key": "asc" }),
+                SortDirection::Desc => serde_json::json!({ "_key": "desc" }),
+            },
```

Comment:
Here it's assumed `_key` sorting in Tantivy matches PostgreSQL's collation. This should be verified for all data types, especially strings with different locales.